inventory/packages/rpm.go

```diff
@@ -25,7 +25,7 @@ import (
 var (
 	rpmquery string
 
-	rpmqueryArgs = []string{"-a", "--queryformat", `%{NAME} %{ARCH} %{VERSION}-%{RELEASE}\n`}
+	rpmqueryArgs = []string{"-a", "--queryformat", "%{NAME} %{ARCH} %{VERSION}-%{RELEASE}\n"}
```

Comment:
What's the difference between `rpmquery` and `rpmQuery`? Should they be the same variable or should one be renamed?

Reply:
command is named rpmquery so I'll stick with that